update i18n for zh_CN

nanhualyq opened the pull request:
#6326

nanhualyq committed:
Update admin.php

nanhualyq committed:
Update conf.php

nanhualyq committed:
Update feedback.php

nanhualyq committed:
Update gen.php

nanhualyq committed:
Update sub.php

nanhualyq committed:
Update conf.php

math-GH committed:
Merge branch 'edge' into patch-1

math-GH commented:
I fixed the conflict, because #6317 renamed `dot paths` => `dot notation`.
@nanhualyq Could you please check that the translation is still valid?

nanhualyq committed:
Update sub.php

nanhualyq commented:
> I fixed the conflict, because #6317 renamed `dot paths` => `dot notation`. @nanhualyq Could you please check that the translation is still valid?

updated!
